src/server/editorServices.ts

```diff
@@ -363,6 +363,11 @@ namespace ts.server {
             this.printProjects();
         }
 
+        private onTypeRootFileChanged(project: ConfiguredProject, fileName: string) {
+            this.updateConfiguredProject(project);

```

Comment:
why not `this.throttledOperations.schedule` ? in case there is a flurry of these, e.g. `npm install`